app/components/Edit/EditSections.js

```diff
@@ -561,16 +553,22 @@ class EditSections extends React.Component {
   }
 
   certifyHAP() {
```

Comment:
This method wasn't being used. The method being used to certifyHAP was called HAPcertify and was in Resource.js, but I moved it here.